pkg/authorization/authz.go

```diff
@@ -85,7 +86,7 @@ func (ctx *Ctx) AuthZRequest(w http.ResponseWriter, r *http.Request) error {
 		}
 
 		if !authRes.Allow {
-			return fmt.Errorf("authorization denied by plugin %s: %s", plugin.Name(), authRes.Msg)
+			return errors.NewErrorWithStatusCode(fmt.Errorf("authorization denied by plugin %s: %s", plugin.Name(), authRes.Msg), http.StatusForbidden)
```

Comment:
Perhaps create a new helper function for this? (`NewRequestForbiddenError`?) Similar to https://github.com/docker/docker/blob/3d13fddd2bc4d679f0eaa68b0be877e5a816ad53/errors/errors.go#L37-L41

(also two other locations may need to be updated then; https://github.com/docker/docker/blob/7fd53f7c711474791ce4292326e0b1dc7d4d6b0f/daemon/network.go#L100 and https://github.com/docker/docker/blob/7fd53f7c711474791ce4292326e0b1dc7d4d6b0f/daemon/network.go#L224)